components/authz-service/server/v2/migration_internal_test.go

```diff
@@ -2279,6 +2294,10 @@ func TestV1PolicyMigration(t *testing.T) {
 			wellknown(t, constants_v1.CSNginxComplianceDataCollectorPolicyID),
 			checks(isSkipped()),
 		},
+		"v1 default application policy": {
```

Comment:
ℹ️ Skipping the default v1 application policy: it's handled by `viewer` and `editor` roles in v2.

Reply:
💡

Reply:
I raised the question to Kyleen, to whom this actually matters :grin:, and she has confirmed it is *not* the same. What they want is "all users to have default access to all [application] things". Right now anyone with a viewer role or on the viewers team has access, but all users are not viewers _by default_. So I think we need a migration for this policy. See https://chefio.slack.com/archives/CFCE1NQ9W/p1560797578017700

Reply:
I'd like to pull in Susan's feedback here. In IAM v2, we don't really follow any concept of "all users having default access" anymore.

Also, if we encourage our customers _not to migrate v1 policies_, we cannot rely on them for any nice out-of-the-box experience when using IAM v2.